src/main/java/lotto/Rank.java

```diff
@@ -0,0 +1,37 @@
+package lotto;
+
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```

언더바를 통해 가독성을 상승시킬 수 있을 것 같아요~

Reply:
아하@.@ 좋은 팁이네요